app/assets/stylesheets/application.css.scss

```diff
@@ -0,0 +1,7 @@
+a {
+  border: none;
+}
+
+div {
+  font-family: "Ricty"
```

Comment:
__[scss_lint]__
* [TrailingSemicolon] Declaration should be terminated by a semicolon
* [StringQuotes] Prefer single quoted strings

<!-- Comment by [SideCI](https://sideci.com) -->